src/main/resources/liquibase/crop_changelog/18_1_0.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="corina" id="v18.1.0-1">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">SELECT count(*) FROM udflds WHERE ftable='ATRIBITS' AND fcode='COLLSITE';</sqlCheck>
```

Comment:
I would add ftype to both queries to ensure there is only one row retrieved. udflds has unique (ftable, ftype, fcode) 
@mchabito

Reply:
Thanks @clarysabel. For some reason, I did not get any notification of your comment even if I'm tagged. 
I will make the change as suggested.